app/frontend/helpers/ui_helper.js

```diff
@@ -23,18 +23,18 @@ class UiHelper {
     this.app_container_height = null;
   }
 
-  configureButtonStyles(context = null) {
+  configureButtonStyles(context=null) {
     if (context == null) {
       context = document;
-    } else {
-      var context = document.querySelector(context);
-    }
+    } else if (typeof context === 'string') {
+      context = document.querySelector(context);
+    } 
   
```

Comment:
Now the `else` case is not accounted for anymore ... we could consider throwing an exception in the else case.